packages/titus-db-manager/lib/routes.js

```diff
@@ -11,13 +11,35 @@ const Migrate = require('../migrate')
 const Truncate = require('../truncate')
 const { version } = require('../package')
 
+const inputSchema = {
+  oneOf: [
+    { type: 'null' },
+    {
+      description: 'Optional schema to process',
+      type: 'object',
+      properties: {
+        schema: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 64
```

Comment:
I wouldn't really put this validation here. let's delegate it to pg, they know how long it can be